lefthook.yml

```diff
@@ -0,0 +1,40 @@
+pre-commit:
+  parallel: true
+  jobs:
+    - name: Format non-markdown
+      glob: "*.{js,json,yml,yaml,css,html}"
+      run: yarn prettier --write --cache {staged_files}
+      stage_fixed: true
+
+    - name: Format markdown
+      glob: "*.md"
+      group:
+        parallel: true
+        jobs:
+          - name: Markdownlint
+            run: yarn markdownlint-cli2 --fix {staged_files}
+            stage_fixed: true
+          - name: Autocorrect
+            run: yarn autocorrect --fix {staged_files}
+            stage_fixed: true
```

Comment:
Could we move autocorrect to a separate rule that only runs on `zh-cn`, see: https://github.com/mdn/translated-content/blob/68dc499b9a2fa51a508df9f866c06aee6601fb64/.autocorrectignore#L6-L7

Reply:
done in https://github.com/mdn/translated-content/pull/26482/commits/31d2eb49013fff33114608e6bb32abbdd16de0e9